runtime/drivers/olap.go

```diff
@@ -741,6 +744,15 @@ func (d Dialect) GetTimeExpr(t time.Time) (bool, string) {
 	}
 }
 
+func (d Dialect) DictGetExpr(lookupTable, lookupColumn, lookupKey string) string {
```

Comment:
nit: Should the function name use "lookup" instead of "dict", e.g. something like `LookupExpression`?